rpcserver.go

```diff
@@ -6701,12 +6707,15 @@ func marshallTopologyChange(topChange *routing.TopologyChange) *lnrpc.GraphTopol
 			},
 			Capacity: int64(channelUpdate.Capacity),
 			RoutingPolicy: &lnrpc.RoutingPolicy{
-				TimeLockDelta:    uint32(channelUpdate.TimeLockDelta),
-				MinHtlc:          int64(channelUpdate.MinHTLC),
-				MaxHtlcMsat:      uint64(channelUpdate.MaxHTLC),
-				FeeBaseMsat:      int64(channelUpdate.BaseFee),
-				FeeRateMilliMsat: int64(channelUpdate.FeeRate),
-				Disabled:         channelUpdate.Disabled,
+				TimeLockDelta:           uint32(channelUpdate.TimeLockDelta),
+				MinHtlc:                 int64(channelUpdate.MinHTLC),
+				MaxHtlcMsat:             uint64(channelUpdate.MaxHTLC),
+				FeeBaseMsat:             int64(channelUpdate.BaseFee),
+				FeeRateMilliMsat:        int64(channelUpdate.FeeRate),
```

Comment:
you can break those this way:
```
				TimeLockDelta: uint32(
					channelUpdate.TimeLockDelta,
				),
				MinHtlc: int64(
					channelUpdate.MinHTLC,
				),
				MaxHtlcMsat: uint64(
					channelUpdate.MaxHTLC,
				),
				FeeBaseMsat: int64(
					channelUpdate.BaseFee,
				),
				FeeRateMilliMsat: int64(
					channelUpdate.FeeRate,
				),
```
otherwise you can disable long-line linting for that block:

```
			//nolint:lll
			RoutingPolicy: &lnrpc.RoutingPolicy{
```